operations/rates.md

```diff
@@ -472,36 +439,142 @@ Note that prices are defined daily, so when the server receives the UTC interval
 
-#### Pricing type
-* `BaseRatePricing`
-* `DependentRatePricing`
-* ...
\ No newline at end of file
+Updates the prices for a given rate. You can make multiple price updates with one API call, and for each one specify the time interval for which the update applies, the price value and the applicable [resource category](resources.md#resource-category). The price will be updated for all service [time units](services.md#time-unit) that the specified time interval intersects. The price is per time unit, e.g. per day or per month. If the resource category `CategoryId` is not specified, the updated price will apply to the base price for all resource categories.
+Note that prices are defined daily, so when the server receives the UTC interval, it first converts it to the enterprise timezone and updates the price on all dates that the interval intersects. Only root rates can be updated (the rates that have no base rate, that have `BaseRateId` set to `null`). It is not permitted to update historical prices older than specified by `EditableHistoryInterval`. Future prices may be updated up to 5 years in the future. The __maximum size of time interval__ is 100 time units or 2 years, whichever is the shorter amount of time.
+
+### Request
+
+`[PlatformAddress]/api/connector/v1/rates/updatePrice`
+
+```javascript
+{
+  "ClientToken": "E0D439EE522F44368DC78E1BFB03710C-D24FB11DBE31D4621C4817E028D9E1D",
+  "AccessToken": "C66EF7B239D24632943D115EDE9CB810-EA00F8FD8294692C940F6B5A8F9453D",
+  "Client": "Sample Client 1.0.0",
+  "RateId": "ed4b660b-19d0-434b-9360-a4de2ea42eda",
+  "PriceUpdates": [
+    {
+      "FirstTimeUnitStartUtc": "2022-01-01T23:00:00.000Z",
+      "LastTimeUnitStartUtc": "2022-01-03T23:00:00.000Z",
+      "Value": 111
+    },
+    {
+      "CategoryId": "e3aa3117-dff0-46b7-b49a-2c0391e70ff9",
+      "FirstTimeUnitStartUtc": "2022-01-04T23:00:00.000Z",
+      "LastTimeUnitStartUtc": "2022-01-05T23:00:00.000Z",
+      "Value": 222
+    }
+  ]
+}
+```
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `ClientToken` | string | required | Token identifying the client application. |
+| `AccessToken` | string | required | Access token of the client application. |
+| `Client` | string | required | Name and version of the client application. |
+| `FirstTimeUnitStartUtc` | string | optional |  |
+| `LastTimeUnitStartUtc` | string | optional |  |
+| `RateId` | string | required | Unique identifier of the base [Rate](rates.md#rate) to update. |
+| `ProductId` | string | optional |  |
+| `PriceUpdates` | array of [Rate price update](rates.md#rate-price-update) | required, max 1000 items | Price updates. |
+| ~~`StartUtc`~~ | ~~string~~ | ~~optional~~ | **Deprecated!** |
+| ~~`EndUtc`~~ | ~~string~~ | ~~optional~~ | **Deprecated!** |
+
+#### Rate price update
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `CategoryId` | string | optional |  |
+| `Value` | number | optional |  |
```

Comment:
Missing description

Reply:
fixed as a part of this [commit](https://github.com/MewsSystems/gitbook-connector-api/pull/635/commits/aea81f3fe83c61522fc3bcfd401a5535459b7ab0)